integration/config/configtx.yaml

```diff
@@ -254,7 +268,7 @@ Orderer: &OrdererDefaults
     # as TLS validation.  The preferred way to specify orderer addresses is now
     # to include the OrdererEndpoints item in your org definition
     Addresses:
-        # - 127.0.0.1:7050
+            #- orderer.example.com:7050
```

Comment:
This is a superfluos difference (and in context of integration config, orderer.example.com doesn't make sense as the only host existing is localhost ..)